gr-digital/examples/equalizers/le_vs_dfe.grc

```diff
@@ -617,7 +623,7 @@ blocks:
     comment: ''
     cons: cons
     maxoutbuf: '0'
-    meas_type: digital.evm_measurement_t_EVM_PERCENT
+    meas_type: digital.evm_measurement_t.EVM_PERCENT
```

Comment:
:+1: We'll probably need to do that in all places that contain `evm_measurement_t_EVM_PERCENT`, right?

Reply:
I believe so. The flowgraph still ran without making any changes to that block, however. GRC auto-generated the proper code when I saved the flowgraph.